main.scss

```diff
@@ -23,7 +23,10 @@
 @import 'scss/wrapped_general';
 
 @if $o-typography-is-silent == false {
-	@include oFontsIncludeAll();
+
+	@if $o-typography-load-fonts == true {
```

Comment:
That would mean there's a possibility, in silent mode, to output the `@font-face` css